docs/user-guide/slicing.ipynb

```diff
@@ -695,6 +700,80 @@
     "## Advanced indexing"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Integer array indexing\n",
```

Comment:
```suggestion
    "### Integer list indexing\n",
```
?
Maybe this makes the distinction with boolean *variable* indexing clearer.

Reply:
I used the name from the numpy docs.